src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -392,7 +398,29 @@ public static void SetWindowPosition(int left, int top)
 
         public static void SetWindowSize(int width, int height)
         {
-            throw new PlatformNotSupportedException();
+            if (Console.IsOutputRedirected)
+                throw new IOException(SR.InvalidOperation_SetWindowSize);
+            if (width <= 0)
+                throw new ArgumentOutOfRangeException(nameof(width), width, SR.ArgumentOutOfRange_NeedPosNum);
+            if (height <= 0)
+                throw new ArgumentOutOfRangeException(nameof(height), height, SR.ArgumentOutOfRange_NeedPosNum);
```

Comment:
These should be moved to Console.SetWindowSize prior to delegating to ConsolePal.SetWindowSize, and then removed from both here and from ConsolePal.Windows.cs.

Reply:
Fixed. Thank you.